serde-reflection/src/de.rs

```diff
@@ -290,12 +278,8 @@ impl<'de, 'a> de::Deserializer<'de> for Deserializer<'de, 'a> {
     where
         V: Visitor<'de>,
     {
-        self.format
-            .unify(Format::Tuple(vec![Format::Unknown; len]))?;
-        let formats = match self.format {
-            Format::Tuple(x) => x,
-            _ => unreachable!(),
-        };
+        let mut formats: Vec<_> = std::iter::repeat_with(Format::unknown).take(len).collect();
```

Comment:
Clever way to rewrite `vec![]`

Reply:
Not sure if this comment shows approval or disapproval :)
To clarify, just in case, `vec![Format::unknown(); len]` would be sharing the same variable `len` times instead of creating `len` distinct variables.

Reply:
Yeah it depends on how Clone is implemented, as it will call clone on the type provided `len` times.

Reply:
Right. Here, we want `Clone` to create sharing.